src/features/imports/Page.tsx

```diff
@@ -248,15 +233,23 @@ export const ImportPage = (props: ImportPageProps) => {
     alert('no issues found!');
   }
 
-  const exportText = state.data.import_metadata.statements != null ? state.data.import_metadata.statements.map((statement) => {
-    const pg = statement.original.split("\n").map(x => `-- ${x}`).join("\n")
-    var crdb = statement.cockroach;
-    crdb.trim();
-    if (crdb.length > 0 && crdb.charAt(crdb.length - 1) !== ';') {
-      crdb += ";";
-    }
-    return '-- postgres:\n' + pg + '\n' + crdb + '\n';
-  }).join('\n') : '';
+  const getExportText = useCallback(() =>
```

Comment:
i'm rereading this and I wonder ... does it make sense to calculate this lazily?
seems expensive to do this every time `statements` changes.
if so, do you mind adding a TODO for me (or i can make it an issue)

Reply:
This is one of those weird React Hooks API quirks that consistently trips me up for a minute. Because this is a call to `useCallback()`, `getExportText` isn't actually the text — its value is an anonymous function!  The export text will be calculated lazily and memoized, and the function itself is what changes when `statements` changes: https://reactjs.org/docs/hooks-reference.html#usecallback